### PR TITLE
NYC 311 Potholes

### PR DESCRIPTION
**Pull request by dirkkelly**
https://citygram-nyc-staging.herokuapp.com/

screen shot 2015-10-17 at 12 28 43 pm

### CODE REVIEW

**Comment by dirkkelly** [comment]
:bowtie: @excepttheweasel how does this look?

File: lib/spy_glass/registry/nyc-311-potholes.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+time_zone = ActiveSupport::TimeZone['Eastern Time (US & Canada)']
+SpyGlass::Registry << SpyGlass::Client::Socrata.new(opts) do |collection|
+  features = collection.map do |item|
+    time = Time.iso8601(item['created_date']).in_time_zone(time_zone).strftime("%m/%d  %I:%M %p")
+    city = item['city'].capitalize
+
+    title =
+      case item['address_type']
+      when 'ADDRESS'
+        "#{time} - #{item['descriptor']} at #{item['incident_address'].titleize} in #{city}."
```
Comment (dirkkelly) [inline lib/spy_glass/registry/nyc-311-potholes.rb]: @excepttheweasel we could probably pull this out of our scripts if we make it into a custom class that people can require and invoke.

Comment (dirkkelly) [inline lib/spy_glass/registry/nyc-311-potholes.rb]: This is the current text on the overall 311, I think it's way too verbose

https://github.com/BetaNYC/citygram-services-nyc/blob/master/lib/spy_glass/registry/nyc-311.rb#L30

screen shot 2015-10-17 at 1 02 43 pm

Comment (j16r) [inline lib/spy_glass/registry/nyc-311-potholes.rb]: Agreed, is that text consistent? perhaps we could pass some details out of it and make it more concise. e.g: "A new 311 case has been opened at" is redundant.

Comment (dirkkelly) [inline lib/spy_glass/registry/nyc-311-potholes.rb]: Yeah, I'm working on a PR to fix the 311 also. I'll do it in this branch.

**Comment by dirkkelly** [comment]
@noneck yo, pr plz

**Comment by noneck** [comment]
@dirkkelly make sure that staging and production are in sync.

**Comment by dirkkelly** [comment]
@noneck roger, wilco

**Comment by dirkkelly** [comment]
`heroku pg:backups restore $(heroku pg:backups public-url --app citygram-staging) DATABASE_URL --app citygram-nyc-staging`

screen shot 2015-10-21 at 7 44 49 pm

**Comment by dirkkelly** [comment]
```
heroku pg:backups restore $(heroku pg:backups public-url --app citygram) DATABASE_URL --app citygram-nyc-staging
```

**Pull request merged**